src/components/Search/Search.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import SearchIcon from "@assets/images/icons/search.svg";
+import "@components/Search/search.scss";
+import { searchBlogPosts } from "@actions/blogActions";
+
+const Search = () => {
+  const dispatch = useDispatch();
+  const query = useSelector((state) => state.search.searchQuery);
+
+  const handleSearch = (event) => {
+    const searchQuery = event.target.value?.toLowerCase();
```

Comment:
Taking the search query in lowercase doesn't seem user-friendly. It would be better to allow users to search using any case they prefer while internally converting the input to lowercase. This ensures that users can search effortlessly without worrying about case sensitivity.

Reply:
Done